copy-reports/README.md

```diff
@@ -0,0 +1,16 @@
+# Copy Reports Script
+
+## Description
+This script automates the migration of specific configuration files from the old Cost Analyzer to the new Aggregator in the Kubecost namespace. It copies files listed in the script from the old pod to the new pod, ensuring a seamless transition.
+
+## Usage
+
+1. Open a terminal.
+
+2. Copy and paste the script into a file, e.g., `copy-reports.sh`.
+
+3. Make the script executable:
+   ```bash
+   chmod +x copy-reports.sh
+   ./copy-reports.sh
```

Comment:
to keep it simpler, you can always just use the shell to execute it as a readable file:

```
bash copy-reports.sh
```
then no need to set the execute bit